test/athens/parser/impl_test.cljc

```diff
@@ -231,10 +231,17 @@
     (utils/parses-to sut/inline-parser->ast
 
                      ;; Any ASCII punctuation character may be backslash-escaped
+                     ;; NOTE: not working in JS environment same as in JVM
```

Comment:
@neotyk would like your input on specialcasing this test. There's another test (a latex one) that is specialcased for cljs too. 

I think this isn't a big issue because the hashtag tests already test that word chars before a hashtag will stop it from being a hashtage.